odoo/tools/misc.py

```diff
@@ -1640,9 +1640,10 @@ def hmac(env, scope, message, hash_function=hashlib.sha256):
     ).hexdigest()
 
 
-ADDRESS_REGEX = re.compile(r'^(.*?)(\s[0-9][0-9\S]*)?( - (.+))?$')
+ADDRESS_REGEX = re.compile(r'^(.*?)(\s[0-9][0-9\S]*)?( - (.+))?$', re.S)
 def street_split(street):
-    results = ADDRESS_REGEX.match(street or '').groups('')
+    rematch = ADDRESS_REGEX.match(street or '')
```

Comment:
This naming is unusual, it gives the impression that we're matching it *again*. `m` or `match` are conventional.

Reply:
renamed to `match`